tools/vscode/README.md

```diff
@@ -0,0 +1,17 @@
+## [License](https://github.com/vmware/differential-datalog/blob/master/LICENSE)
```

Comment:
We avoid adding binary files to the repository.
So instead of adding the vsix files, you should add the source files that you used to produce the vsix files, and a script that produces the vsix files from the sources. Then the source files can also be reviewed.

Reply:
Added source code 
https://github.com/vmware/differential-datalog/pull/829